A6-client-retries.md

```diff
@@ -116,6 +116,14 @@ When gRPC receives a non-OK response status from a server, this status is checke
 
 In general, only status codes that indicate the service did not process the request should be retried. However, a more aggressive set of parameters can be specified when the service owner knows that a method is idempotent, or safe to be processed more than once by the server. For example, if an RPC to a delete user method fails with an `INTERNAL` error code, it's possible that the user was already deleted before the failure occurred. If the method is idempotent then retrying the call should have no adverse effects. However, it is entirely possible to implement a non-idempotent delete method that has adverse side effects if called multiple times, and such a method should not be retried unless the error codes guarantee the original RPC was not processed. It is up to the service owner to pick the correct set of retryable status codes given the semantics of their service. gRPC retries do not provide a mechanism to specifically mark a method as idempotent.
 
+##### Validation of retryPolicy
+
+If `retryPolicy` is specified in a service config choice, the following validation rules apply:
+1. `maxAttempts` should be specified and should be a JSON integer value greater than 1.
```

Comment:
Please replace the word "should" with "MUST" throughout this new text.

Reply:
done